akka-http/src/main/scala/akka/http/scaladsl/server/directives/FormFieldDirectives.scala

```diff
@@ -217,7 +217,7 @@ object FormFieldDirectives extends FormFieldDirectives {
     import akka.http.scaladsl.server.util.TupleOps._
 
     implicit def forTuple[T](implicit fold: FoldLeft[Directive0, T, ConvertFieldDefAndConcatenate.type]): FieldDefAux[T, fold.Out] =
-      fieldDef[T, fold.Out](fold(pass, _))
+      fieldDef[T, fold.Out](fold(toStrictEntity(StrictForm.toStrictTimeout), _))
```

Comment:
Why exactly here? I guess this helps for the `formField('a, 'b)` case. If we would put it in the directive directly, wouldn't it then also work for the nested formField directive tasks?

Reply:
Yeah, that would indeed also solve scenario `B`, at the expense of also inserting a `toStrict` in situations where it's not strictly necessary. Might be fine.